ios_02_simplecalculator_Uri/Controller/ViewController.swift

```diff
@@ -26,182 +26,31 @@ class ViewController: UIViewController {
     @IBOutlet weak var btnMes: UIButton!
     @IBOutlet weak var btnMenos: UIButton!
     @IBOutlet weak var btnAC: UIButton!
-    
     @IBOutlet weak var btnTantPercent: UIButton!
     @IBOutlet weak var btnMasMenos: UIButton!
     @IBOutlet weak var lblDisplay: UITextField!
     
-    private var display = "0"
-    private var valor1: Double?
-    private var valor2: Double?
-    private var result: Double?
     //var for know if the calculator allready have any value
-    private var statusInit = false
-    private var currentOp = ""
-    private var currentVar = "val1"
-    
     let defaults = UserDefaults.standard
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        btnDecimals.isHidden = defaults.bool(forKey: "decimalsEnabled")     //Hide decimal button by user settings
-        lblDisplay.text = defaults.string(forKey: "LastResult")             //Display last results
+        //btnDecimals.isHidden = defaults.bool(forKey: "decimalsEnabled")     //Hide decimal button by user settings
+        //lblDisplay.text = defaults.string(forKey: "LastResult")             //Display last results
     }
```

Comment:
try to not have any commented code in a main pull request :)

Reply:
Es para la siguiente feature